perma_web/perma/tasks.py

```diff
@@ -973,33 +948,29 @@ def _proxy_request(self):
             print("Finished fetching URL.")
 
             # Get a fresh copy of the page's metadata, if possible
-            with browser_running(link, browser):
-                inc_progress(capture_job, 1, "Retrieving page metadata (after onload)")
-                add_thread(thread_list, get_metadata, args=(
-                    page_metadata,
-                    link,
-                    browser
-                ))
+            dom_tree = get_dom_tree(browser)
+            get_metadata(page_metadata, dom_tree)
```

Comment:
What would you think of waiting until after the scroll for this second load? I wonder if on average that would that make the captured DOM even better, or would just mean the capture process fails out more often before we get to it....